core/templates/dev/head/domain/summary/exploration-summary-backend-api.service.ts

```diff
@@ -36,16 +36,19 @@ angular.module('oppia').factory('ExplorationSummaryBackendApiService', [
         for (var i = 0; i < explorationIds.length; i++) {
           returnValue.push(null);
         }
-        return $q.resolve(returnValue);
+
+        if (errorCallback) {
+          errorCallback(returnValue);
+        }
+        return;
       }
 
       var explorationSummaryDataUrl = EXPLORATION_SUMMARY_DATA_URL_TEMPLATE;
 
       $http.get(explorationSummaryDataUrl, {
         params: {
-          stringified_exp_ids: JSON.stringify(explorationIds),
```

Comment:
Why was this changed? Isn't explorationIds an array, and shouldn't it be converted to a string here?

Reply:
I just fixed it. Done.